src/@types/types.d.ts

```
@@ -27,3 +27,17 @@ export interface SharingEntry {
   seq: number;
   event: any;
 }
+
+declare global {
+  interface Window {
+    /** Injected by hkbus.app app webview
+     * @since Dec 2024
+     */
+    systemColorSchemeCallbacks?: Function[];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)